src/sentry/utils/snuba.py

```diff
@@ -1009,30 +1008,25 @@ def _bulk_snuba_query(
 RawResult = tuple[urllib3.response.HTTPResponse, Callable[[Any], Any], Callable[[Any], Any]]
 
 
-def _snql_query(
-    params: tuple[
-        tuple[SnubaQuery, Callable[[Any], Any], Callable[[Any], Any]], Hub, Mapping[str, str], str
-    ]
-) -> RawResult:
+def _snql_query(params: tuple[RequestQueryBody, Hub, Mapping[str, str], str]) -> RawResult:
```

Comment:
we had other sentry files using this function, but I verified they all conform to the changed input spec

Reply:
why are other people able to call these functions? are able to / should we be defining our modules to only expose the public functions

Reply:
oh actually the only use of this function is within tests...